templates/dashboard/contact_detail.html

```diff
@@ -70,4 +72,37 @@ <h1 class="h3">
   </div>
 {% endblock %}
 
+{% block script %}
+    {% if current_user.is_premium() %}
+    <script>
+    const MAX_BYTES = 2048; // 2KiB
```

Comment:
Some keys are quite big (mine is 3K), to be on the safe side I think we can use a bigger value here, for example 10K.

Reply:
Increased to 10K